docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:latest
```

Comment:
Maybe we should pin this to a particular version so that we have a specific working configuration.